test/integration/pointers_incomplete.toml

```diff
@@ -42,7 +43,7 @@ definitions = '''
       "NOT": "members"
     }]'''
   [cases.raw_null]
-    skip = "codegen fails on this"
+    oid_skip = "oid codegen fails on this" # https://github.com/facebookexperimental/object-introspection/issues/17
```

Comment:
Don't oid and oil share codegen? How can one work and not the other?

Reply:
(I see this is what's happening, just confused!)

Reply:
Yep, it's weird. The only bit that I would expect to be different is [the entrypoint generation](https://github.com/facebookexperimental/object-introspection/blob/main/src/OICodeGen.cpp#L3190-L3207), but it doesn't look like the code gets there. Didn't get to the bottom of it so opened the issue for now.